deploy/images/controller/bicep/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
+
+# Install tools needed  
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        curl \
+        ca-certificates 
+        
+WORKDIR /
+
+RUN curl -Lo bicep https://github.com/Azure/bicep/releases/latest/download/bicep-linux-arm64 \
```

Comment:
The architecture should be dynamically decided depending on the host machine. There will be multiple architectures published for the image itself, so the binary needs to be compatible